ui/src/qlcplus_de_DE.ts

```diff
@@ -5734,7 +5734,7 @@ Dauer: %3
         <location filename="virtualconsole/vccuelistproperties.ui" line="276"/>
         <source>Side Fader</source>
         <oldsource>Side Faders</oldsource>
-        <translation type="unfinished">Seiten-Schieberegler</translation>
+        <translation>Seiten-Schieberegler</translation>
```

Comment:
"Side Faders" translated to "Seiten-Schieberegler".
Stick to one, preferrably the first one.